examples/tfengine/generated/devops/devops/variables.tf

```diff
@@ -16,7 +16,10 @@ variable "admins_group" {
   type = object({
     id = string
   })
-  description = "Group which will be given admin access to the folder or organization."
+  description = <<EOF
+Group which will be given admin access to the folder or organization.
+It will be created if 'exists' is false.
```

Comment:
This sentence is confusing as the `exists` is not a var here. Maybe remove it from the schema description as well.

Reply:
This is something I mentioned with @MartinPetkov on my very first Pull Request. There are descriptions that should be slightly different from the rendered version, but we agreed to keep it like that.

We can go deep on that discussion here. Personally, I think it can be removed for this case since the `exists` schema property is already documented, but there might be other cases where we shouldn't remove something from the schema.

Reply:
Yea, I think removing in this case is probably fine. In other cases, we might need to develop some other mechanism to include only part of the description from the schema.

Reply:
Great, removed it and updated.

I'm resolving